secrethub_wrapper.go

```diff
@@ -34,10 +34,20 @@ import (
 	"github.com/secrethub/secrethub-go/pkg/secrethub"
 )
 
+func getVersion() []secrethub.ClientOption {
+	options := []secrethub.ClientOption{
+		secrethub.WithAppInfo(&secrethub.AppInfo{
+			Name:    "secrethub-xgo",
+			Version: "1.0.0",
+		}),
+	}
+	return options
+}
```

Comment:
I don't think calling this function `getVersion` is descriptive of what it does. And since it is always used like this:
```
secrethub.NewClient(getVersion()...)
```
We could make a `Client()` function, which creates and returns a new SecretHub client created with the desired client options.

Reply:
Sounds like a clearer idea. Will do that.